### ISSUE TITLE
View Note/Save Note Inconsistent 

### ISSUE THREAD

**Issue by edwardwptc** [TwinePlatform/twine-monolith#628]
Volunteer creates a log, adds a note
Saves log
"Saved" appears
Continue
Click Time
Find log
There is no "View note" button to indicate a note exists
Click Edit log
Click Edit Note
Note is blank

And

Login as admin
View same log
No "View Note" button
Click Edit Log
Click Edit Note
Add note
Go back
Edit Time
"Saved" appears, continue
Log continues to have no note

Tested on volunteer and admin, as you can tell. It appears that my inability to see the "View Note" feature is because the saving of notes is extremely inconsistent.

**Comment by King-Mob** [comment]
Yes, i've just added this as a bug. The problem is with the API, for some reason the security isn't authorising volunteers to add notes. It's on the project board
